clang/lib/Sema/SemaDecl.cpp

```diff
@@ -11465,6 +11465,10 @@ static bool CheckMultiVersionFirstFunction(Sema &S, FunctionDecl *FD) {
   // otherwise it is treated as a normal function.
   if (TA && !TA->isDefaultVersion())
     return false;
+  // The target_version attribute only causes Multiversioning if this
+  // declaration is NOT the default version.
+  if (TVA && TVA->isDefaultVersion())
+    return false;
```

Comment:
```suggestion
  if (!TVA || TVA->isDefaultVersion())
    return false;
```

Reply:
Hmm, we can't write it this way I am affraid because the function used both for target and target_version attributes. If TA is not null and TVA is null we are early exiting while we shouldn't.